src/http/extractor.rs

```diff
@@ -167,10 +171,15 @@ impl FromRequest for AuthUser {
 }
 
 #[async_trait]
-impl FromRequest for MaybeAuthUser {
+impl<B> FromRequest<B> for MaybeAuthUser
+where
+    B: http_body::Body + Send,
+    B::Data: Send,
+    B::Error: Into<BoxError>,
```

Comment:
Same here. The body isn't consumed so these bounds shouldn't be necessary.

Reply:
Removed unnecessary bounds